lms/djangoapps/verify_student/models.py

```diff
@@ -228,10 +225,8 @@ def verification_valid_or_pending(cls, user, earliest_allowed_date=None, window=
             user:
             earliest_allowed_date: earliest allowed date given in the
                 settings
-            window: If window=None, this will check for the user's
+            This will check for the user's
```

Comment:
Remove line#228-229 (If necessary then add this line to `verification_valid_or_pending` method's main docstring).